addons/point_of_sale/static/src/js/Popups/MoneyDetailsPopup.js

```diff
@@ -0,0 +1,80 @@
+odoo.define('point_of_sale.MoneyDetailsPopup', function(require) {
+    'use strict';
+
+    const { useState, useRef } = owl.hooks;
+    const PosComponent = require('point_of_sale.PosComponent');
+    const Registries = require('point_of_sale.Registries');
+
+    /**
+     * Even if this component has a "confirm and cancel"-like buttons, this should not be an AbstractAwaitablePopup.
+     * We currently cannot show two popups at the same time, what we do is mount this component with its parent
+     * and hide it with some css. The confirm button will just trigger an event to the parent.
+     */
+    class MoneyDetailsPopup extends PosComponent {
+        constructor() {
+            super(...arguments);
+            this.currency = this.env.pos.currency;
+            this.state = useState({
+                moneyDetails: {'0.05': 0, '0.10': 0, '0.20': 0, '0.50': 0, '1.00': 0, '2.00': 0,
```

Comment:
This Money detail is good for EUR, but doesn't work for other currencies.

Reply:
It's a WIP, the `moneyDetails` is hard coded at the moment but will be a data fetched from the `pos.config`